egs2/l3das22/enh1/local/l3das22_data_prep.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+
+. ./path.sh
+
+help_message=$(cat << EOF
+Usage: $0 [L3DAS path]
+  required argument:
+    L3DAS path: path to the L3DAS directory
+    NOTE:
+        You can download it manually from
+            https://www.kaggle.com/datasets/l3dasteam/l3das22
+EOF
+)
+
+
+if [ $# -ne 1 ]; then
+  echo "${help_message}"
+  exit 1;
+fi
+
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+L3DAS22=$1
+
+tmpdir=data/temp
```

Comment:
`mktemp` would be better in the case, and please also use trap to delete the corresponding folder after the preparation.

Reply:
Since the files under the temp directory are used in the following recipe (local/l3das22_multi_channel.sh), do you think we could keep using this directory for passing the files? (or we could move mktemp to local/data.sh)